modules/administration-guide/pages/configuring-advanced-authorization.adoc

```diff
@@ -0,0 +1,54 @@
+:_content-type: PROCEDURE
+:description: Configuring advanced authorization
+:keywords: authorization, user, group
+:navtitle: Configuring advanced authorization
+// :page-aliases:
+
+[id="configuring-advanced-authorization"]
+= Configuring advanced authorization
+
+You can determine which users and groups are allowed to access {prod-short}.
```

Comment:
@tolusha Should we clarify that this is an OpenShift feature, or this would work on vanilla k8s as well?